ram.tf

```diff
@@ -2,7 +2,6 @@ resource "aws_ram_resource_share" "endpoint_share" {
   count                     = length(var.resource_share_accounts) > 0 ? 1 : 0
   name                      = "route53-${var.forward_domain}-share"
   allow_external_principals = false
-  #tags                      = merge(local.common_tags, var.additional_tags)
```

Comment:
[It allows tags](https://www.terraform.io/docs/providers/aws/r/ram_resource_share.html). I feel like "tag everything" is a good rule to follow but I also like breaking the rules.